### ISSUE TITLE
About the exercise

### ISSUE THREAD

**Issue by c31601** [2017-fall-DL-training-program/PyTorchWarmUp#3]
Hi Assistants,

Could we change the layer, such ReLU -> PReLU?
Or we could only modify the hyper-parameters, such as iteration, lr...etc.?

The score of the exercise only depends on performance of the Res110?
Res20 & Res56 not considered?

Thank you so much!

**Comment by JiaRenChang** [comment]
Hi,
Does you mean the exercise in the WarmUp?
The exercise in the **WarmUp** is just for your practice, and you don't have to submit your result.
----------
If you want to ask the detail in **homwork1** ,please move to the **ConvNetwork** repository.
https://github.com/2017-fall-DL-training-program/ConvNetwork

Jia-Ren

**Comment by c31601** [comment]
Thanks, I will transfer to convnetwork
